shell/platform/linux/fl_settings_plugin.cc

```diff
@@ -32,6 +33,27 @@ struct _FlSettingsPlugin {
 
 G_DEFINE_TYPE(FlSettingsPlugin, fl_settings_plugin, G_TYPE_OBJECT)
 
+static bool is_dark_color(GdkRGBA* color) {
+  return color->red < 0.5 && color->green < 0.5 && color->blue < 0.5;
+}
```

Comment:
We have some very similar logic in the framework. Please take a look:
https://github.com/flutter/flutter/blob/8fe4cc79648a952f9c7e49a5248756c2ff98fa3b/packages/flutter/lib/src/material/theme_data.dart#L1470-L1488

Reply:
Nice, thanks! :+1: